Userland/Libraries/LibC/fenv.cpp

```diff
@@ -46,6 +49,161 @@ static void set_mxcsr(u32 new_mxcsr)
 }
 
 static constexpr u32 default_mxcsr_value = 0x1f80;
+
+#elif ARCH(RISCV64)
+
+// Table 11.1 (frm rounding mode encoding)
```

Comment:
can you add the document name to the table/figure references in the cpp file? 

I assume these are for `RISC-V F extension version 2.2` , same as fenv_t in the header?